client/src/questions-n-answers/sub-components/mini-components/userQuestion.jsx

```diff
@@ -31,6 +32,11 @@ class UserQuestion extends React.Component {
         this.props.updateQuestions()
 
       })
+    this.setState({
+      yourQuestion: '',
```

Comment:
Think the placeholder css is definitely a step up from preloaded state, good call